src/cdk/text-field/autosize.ts

```diff
@@ -35,10 +36,16 @@ import {fromEvent, Subject} from 'rxjs';
 export class CdkTextareaAutosize implements AfterViewInit, DoCheck, OnDestroy {
   /** Keep track of the previous textarea value to avoid resizing when the value hasn't changed. */
   private _previousValue: string;
+  private _initialHeight: string | null;
   private readonly _destroyed = new Subject<void>();
 
   private _minRows: number;
   private _maxRows: number;
+  private _enabled: boolean = true;
+
+  private get textarea(): HTMLTextAreaElement {
```

Comment:
Instead of this, change the constructor:

```ts
constructor(elementRef, ...) {
  this._textareaElement = elementRef.nativeElement as HTMLTextAreaElement;
}
```

Reply:
Good idea and changed. Doesn't really make to big of a difference in the transpiled js because the cast is removed anyways, but yea, way cleaner this way.